src/compiler/scala/tools/reflect/FormatInterpolator.scala

```diff
@@ -94,6 +98,7 @@ abstract class FormatInterpolator {
             else all.head + all.tail.map { case req(what) => what case _ => "?" }.mkString(", ", ", ", "")
           }
           c.error(args(argi).pos, msg)
+          reported = true
```

Comment:
Could `reported` be a local variable next to `formatting`? It's not obvious when instances of `FormatInterpolator` are created.

There are also some invocations of `c.error` in `escapeHatch` - does that matter?

Reply:
The cached FastTrack mechanism results in a `ctx => (new { val c = ctx } with FastStringInterpolator).interpolateF`, so a new interpolator per invocation. Not obvious.

`reported` is set also by `Conversion#errorAt`, so that is why it's not local. The flag is needed so that it doesn't attempt a bad compiletime format, so the bad escapes in `escapeHatch` should matter. I'll make a test.